test/test_e2e_asr_transformer.py

```diff
@@ -58,13 +58,19 @@ def make_arg(**kwargs):
         dunits=16,
         sym_space="<space>",
         sym_blank="<blank>",
+        transformer_decoder_selfattn_layer_type="selfattn",
+        transformer_encoder_selfattn_layer_type="selfattn",
         transformer_init="pytorch",
         transformer_input_layer="conv2d",
         transformer_length_normalized_loss=True,
         report_cer=False,
         report_wer=False,
         mtlalpha=0.0,
         lsm_weight=0.001,
+        wshare=4,
+        ldconv_encoder_kernel_length="21_23_25_27_29_31_33_35_37_39_41_43",
+        ldconv_decoder_kernel_length="11_13_15_17_19_21",
+        ldconv_usebias=False,
```

Comment:
To increase coverage, use kwargs to test ldconv because this change ignores cases without ldconv

Reply:
https://github.com/espnet/espnet/blob/f94ba48f6dbf46428ae22d2c85a205dae71e4d62/test/test_e2e_asr_transformer.py#L130-L142
Here you can add these args in the model_dict parameter like
```python
ldconv_args = dict(
        transformer_decoder_selfattn_layer_type="selfattn",
        transformer_encoder_selfattn_layer_type="selfattn",
        wshare=4,
        ldconv_encoder_kernel_length="21_23_25_27_29_31_33_35_37_39_41_43",
        ldconv_decoder_kernel_length="11_13_15_17_19_21",
        ldconv_usebias=False,
)

 @pytest.mark.parametrize( 
     "module, model_dict", [ 
         ('pytorch', {}), 
         ...
         ('pytorch', ldconv_args), 
         ('chainer', {}), 
     ] 
 ) 
```

Reply:
Sure, I added parameters for all the layer type.